File: Java Program/Swap.java
import java.util.Scanner;
public class Swap
{
  public static void main(String[] args) 
  {
    int a,b;
    Scanner sc=new Scanner(System.in);
    System.out.println("Enter two number ");
    a=sc.nextInt();
    b=sc.nextInt();
    System.out.println("BEFORE SWAP");
    System.out.println("a="+a);
    System.out.println("b="+b);
    a=a+b;
    b=a-b;
    a=a-b;
    System.out.println("AFTER SWAP ");
    System.out.println("a="+a);
    System.out.println("b="+b);

    sc.close();
  }
}

File: C Program/swap.c
#include<stdio.h>

int main()
{
    int a,b,c;
    printf("enter two number ");
    scanf("%d %d",&a,&b);
    printf("a=%d \tb=%d",a,b);
    c=a;
    a=b;
    b=c;
    printf("\na=%d \tb=%d",a,b);
    return 0;
}